odoo_module_migrate/migration.py

```diff
@@ -89,9 +90,24 @@ def __init__(
         for module_name in module_names:
             self._module_migrations.append(ModuleMigration(self, module_name))
 
+        if os.path.exists(".pre-commit-config.yaml"):
+            self._run_pre_commit(module_names)
```

Comment:
maybe add an option for that point could avoid to force users to run pre-commit, if they don't want for any reason. could be default to True however.

don't you think ?

Reply:
Ok